cdap-docs/developers-manual/source/building-blocks/workers.rst

```diff
@@ -0,0 +1,82 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _workers:
+
+============================================
+Workers
+============================================
```

Comment:
These over and underlines can be shortened to just the length of the enclosed text.